pages/common/wget.md

```diff
@@ -11,26 +11,26 @@
 
 `wget -O {{bar}} {{https://example.com/foo}}`
 
-- Download a single web page and all its resources (scripts, stylesheets, images, etc.):
+- Download a single web page and all its resources with 3-second intervals between requests (scripts, stylesheets, images, etc.):
 
-`wget --page-requisites --convert-links {{https://example.com/somepage.html}}`
-
-- Download a full website, with 3-second intervals between requests:
-
-`wget --mirror --page-requisites --convert-links --wait=3 {{https://example.com}}`
+`wget --page-requisites --convert-links --wait=3 {{https://example.com/somepage.html}}`
 
 - Download all listed files within a directory and its sub-directories (does not download embedded page elements):
 
 `wget --mirror --no-parent {{https://example.com/somepath/}}`
 
-- Download the contents of an URL via authenticated FTP:
+- Limit the download speed with specific number of retry:
+
+`wget --limit-rate={{300k}} -tries={{100}} {{https://example.com/somepath/}}`
+
+- Download a file from a password protected site and FTP:
```

Comment:
This does not make much sense. FTP and HTTP Basic Auth are two different things and have different command-line options for user and password. You should pick one between the two (I'd suggest keeping FTP as it is more common).

Reply:
Specify the user and assword work for both FTP and HTTP file. These parameters can be overridden using the --ftp-user and --ftp-password options for FTP connections and the --http-user and --http-password options for HTTP connections.

Reply:
Oh, if those parameters are the same then it's cool. Should probably reword the example description though. Check my new comment below.